packages/@o3r/core/schematics/add-modules/schema.json

```diff
@@ -4,5 +4,15 @@
   "title": "Add available Otter modules",
   "description": "ng generate Add module",
   "properties": {
+    "preset": {
+      "type": "string",
+      "description": "Preset of module list to automatically install",
+      "default": "",
+      "enum": [
+        "",
```

Comment:
could be none instead of empty string to be clearer ?

Reply:
Not sure, it is convenient in the way to have a nullish default value